inginious/frontend/task_dispensers/__init__.py

```diff
@@ -12,6 +12,11 @@ def __init__(self, task_list_func, dispenser_data, database, course_id):
         """
         pass
 
+    @abstractmethod
+    def get_stored_submissions(self, taskid):
```

Comment:
As it is being modified, I think it would be more intuitive and readable for newcomers to rename ``stored_submissions`` to ``no_stored_submissions`` (no = number abbreviation).